redis/src/lib.rs

```diff
@@ -1,5 +1,9 @@
 //! Redis support for the `bb8` connection pool.
 //!
+//! _Due to the default connection config, pooled connections have a response timeout and can't be used for blocking commands._ \
+//! _See [`get_multiplexed_async_connection`](redis::Client::get_multiplexed_async_connection) and [`AsyncConnectionConfig`](redis::AsyncConnectionConfig) for more details._
```

Comment:
Okay, I think I finally understand what the issue is that you're talking about. So did they change this in a recent release?

I guess my beef with this is that it seems in no way to specific to connection pooling, so I don't see why we should document it.